backend-styleguide/README.md

```diff
@@ -313,6 +313,53 @@ enumerize :gender, in: %w[
 ], scope: true
 ```
 
+### 👉 Use comments to define tables and fields
+
+We use comments to define tables and fields. This is a good approach to help newcomers and others Geeks.
```

Comment:
```suggestion
We use comments to define tables and fields. This is a good approach to help newcomers and other Geeks.
```

Reply:
fixed in https://github.com/GeekHunter-Brasil/hello-world/pull/12/commits/e885b0aa55ee15347b80ec92ff29b8740c957e66